staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -52,9 +52,12 @@ func main() {
 	arguments.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt")
 
 	// Custom args.
+	// TODO: make callers pass this in.  It is too opaque here, and any use of
```

Comment:
yes, please!

Reply:
should be easy - just a Makefile change.  Followup PR :)

Reply:
hell, I'll just add a commit here

Reply:
actually I wonder whether we shouldn't leave the apimachinery ones in the generator, and only move the kube specific out. our generators are not kube specific, but they are apimachinery specific. And I don't like to tell everybody to updates his/her scripts when we change apimachinery.

Reply:
I'm happy to back out this commit and debate it separately.  The problem is that "extra" isn't treated as extra - it's EITHER the commandline OR the default, but not both.  We could fix that, of course.